readme.md

```diff
@@ -135,25 +118,19 @@ $ npm -v
 
 * **Question:** What is `path` and where does it come from?
 
-* **Your Answer:**
-
----
+* **Your Answer:** It's the path/folder structure of the file. It comes from the node module by default.
```

Comment:
Strictly speaking this is not right. `path` refers to a "package" that you get for free with Node. You can use different _methods_ on `path` (e.g. `path.resolve()`) to get the file name/structure.